harness/tests/experiment/keras/test_tf_keras_trial.py

```diff
@@ -365,6 +365,131 @@ def make_workloads() -> workload.Stream:
         controller.run()
 
 
+@pytest.mark.tensorflow
+@pytest.mark.gpu
```

Comment:
does cifar10 need gpu testing?  Why can't we run the cifar model against just CPUs and call it a day?